### DISCUSSION TITLE
Pages `build` failed: `Billing owner (User or Organization) is deleted in GitHub.`

### DISCUSSION THREAD

**Discussion by luphoria** [community/community#40113] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Just set up a basic gh-pages site to attempt to redirect the user to the main website (SEO), but gets the following error on build:
`Billing owner (User or Organization) is deleted in GitHub.`
First assumed there was a Pages conflict.. and there was one!... but it's since been deleted, and the issue is still present.

**Comment by luphoria** [comment]
see: https://github.com/titaniumnetwork-dev/gh-pages-legacy-redirect/actions/runs/3551018351

**Comment by ossanna16** [comment]
Hi there @luphoria and welcome to our community! Thank you for reporting this bug.

**Comment by SaidRasinlic** [comment]
Same happens to me!

**Comment by luphoria** [comment]
We ended up giving up on it, but please keep this open for others affected ( @SaidRasinlic )

**Comment by SaidRasinlic** [comment, marked as answer]
#### Shortly: One of the options is to rebuild your project by pushing an empty commit to your branch just to trigger a rebuild for github-pages, and if that doesn't work, there is nothing else you can do except open a new ticket and send it to the GitHub Support team and they will respond within 1 day! To send a ticket, please follow this link: https://support.github.com/

@luphoria The GitHub Support team fixed this for me! I had this problem because I deleted my GitHub profile, then changed my mind and asked them to recover it; I was lucky, and they did (which is not always the case; be careful once you decide to delete it); the problem most likely occurred as a result of this; once you delete your account, GitHub stops billing you, so the best thing to do is send them a ticket regarding this problem.

**Reply by luphoria** [reply]
Very interesting, and I think you're right!
Hopefully this gets fixed so we future folks don't have to contact through support to get this fixed.
